src/app/organization/employees/view-employee/view-employee.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="mat-body-strong">
+          First Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ employeeData.firstname }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Last Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ employeeData.lastname }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Office
+        </div>
+
+        <div fxFlex="50%">
+          {{ employeeData.officeName }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Is Loan Officer
+        </div>
+
+        <div fxFlex="50%" [className]="employeeData.isLoanOfficer === true ? 'true' : 'false'">
```

Comment:
```suggestion
        <div fxFlex="50%">
```
No class is being applied on the basis of this condition.